src/app/bisoncoin/src/App.vue

```diff
@@ -1,22 +1,24 @@
 <template>
   <div id="app">
-    <LoginPage msg="Welcome to BisonCoin!" />
+    <router-view />
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
 import LoginPage from "./pages/LoginPage.vue";
+import HomePage from "./pages/HomePage.vue";
 
 @Component({
   components: {
```

Comment:
I don't think we need these components anymore since we are not adding them to the App template

Reply:
True! Updated